src/dir.cr

```diff
@@ -200,6 +203,11 @@ class Dir
     0
   end
 
+  # Creates a new directory at the given path. The linux-style permission mode
```

Comment:
Please mention that it created intermediates directories in the first sentence, because that's what appears in the short summary of the method.